### PR TITLE
Protocol fees buy back and burn

### PR DESCRIPTION
**Pull request by julien51**
Adding details on the protocol fee as well as the projected swap and burn mechanism.

### CODE REVIEW

File: docs/governance/unlock-dao-tokens.md [hunk 1]
```diff
@@ -46,3 +46,11 @@ This design achieves the following characteristics:
 - Higher value transactions yield a larger amount of UDT tokens, everything else being equal,
 - Earlier transactions yield a larger amount of UDT tokens, everything else being equal,
 - The cost of buying UDT on a decentralized exchange is always lower than the cost of sending "fake" transactions to the network in order to collect UDT.
+
+## Protocol Fee
+
+As of Unlock version 13, the protocol includes a "fee switch" but the fee is currentky set to be 0%. This value could be changed by [DAO members](./unlock-dao.md) through an onchain proposal and vote.
+
+If the fee amount is larger than 0%, on every transaction for which there is a payment ([`purchase`](/core-protocol/smart-contracts-api/PublicLock#purchase), [`extend`](/core-protocol/smart-contracts-api/PublicLock#extend) or [`renewMembershipFor`](/core-protocol/smart-contracts-api/PublicLock#renewmembershipfor)), the lock contract will send a portion of the payment to the Unlock contract.
```
Comment (ccarfi) [inline docs/governance/unlock-dao-tokens.md:54]: is this the "Unlock contract v13" itself? or the timelock contract? or a different contract?

File: docs/governance/unlock-dao-tokens.md [hunk 2]
```diff
@@ -46,3 +46,11 @@ This design achieves the following characteristics:
...
+
+The Unlock Labs team will submit a DAO proposal to add a mechanism that lets anyone submit a transaction to the Unlock contract in order to swap some of the fees collected into UDT tokens and burn these tokens.
```
Comment (ccarfi) [inline docs/governance/unlock-dao-tokens.md:56]: how about:

"in the future, the Unlock Labs team may submit a DAO proposal..."

also, same question as upthread on the "unlock contract" being referred to here.

**Review by ccarfi — COMMENTED** [review]
@julien51 i asked a couple of questions on this

File: docs/governance/unlock-dao-tokens.md (hunk 1 above)
Comment (julien51) [reply docs/governance/unlock-dao-tokens.md:54]: Good question, it is in fact Unlock v12!

File: docs/governance/unlock-dao-tokens.md (hunk 2 above)
Comment (julien51) [reply docs/governance/unlock-dao-tokens.md:56]: It is the actual Unlock contract. I will add links to make it more obvious.

**Review by ccarfi — APPROVED** [review]

**Pull request merged**